build-phar.sh

```diff
@@ -1,11 +1,19 @@
 #!/usr/bin/env bash
 
+set -euf -o pipefail
+
 ./download-box.sh
 
+function restorePlatform {
```

Comment:
Hmm, this is a rare signature for bash functions IMO.

Reply:
wdym?

Reply:
Usually bash functions are written as such:
```bash
functionName() {
    body
}
```

Reply:
Ah right, it's been a while since I've done some bash so I just went with whatever though was correct and my IDE didn't complain either... Can be renamed I guess